Atomic/install.py

```diff
@@ -59,6 +59,9 @@ def cli(subparser):
                 system_xor_user.add_argument("--system", dest="system",
                                              action='store_true', default=False,
                                              help=_('install a system container'))
+                installp.add_argument("---generate-rpm", dest="generate_rpm",
```

Comment:
I would prefer that the user not need to specify this.  This should happen automatically if the image supports generation of rpms for install.